Assets/General/Scripts/Console/FreeLookCam/BoatScriptTag.cs

```diff
@@ -0,0 +1,11 @@
+/*****************************************************************************
+// Name: BoatScriptTag.CS
+// Author: Nabil Tagba
+// Overview: Acts as a safe take to get the boat
+*****************************************************************************/
+using UnityEngine;
+
+public class BoatScriptTag : MonoBehaviour
```

Comment:
We probably shouldn't even have this script if it doesn't do anything

Reply:
^ I think the idea is to find the object by having the component but I imagine there are better ways to do this